walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -28,45 +27,46 @@
     }
     
     for(Event event: events){
-        System.out.print(checkAttending(event,request));
         if(checkAttending(event, request)){
             eventsArrayList.add(event);
         }
     }
 
-    if(request.getAttendees().size() == 0 || eventsArrayList.size() == 0){
+    if(eventsArrayList.size() == 0){
         meetingTimes.add(TimeRange.WHOLE_DAY);
         return meetingTimes;
     }
 
-    Event[] eventsArray = eventsArrayList.toArray(new Event[eventsArrayList.size()]);
-    int latestEndTime = eventsArray[0].getWhen().end();
-    for(int i = 0; i < eventsArray.length; ++i){
-        if(i == 0){
-            int firstEventStart = eventsArray[i].getWhen().start();
-            if((firstEventStart - TimeRange.START_OF_DAY) >= request.getDuration()){
-                meetingTimes.add(TimeRange.fromStartEnd(TimeRange.START_OF_DAY, firstEventStart, false));
-            }
+    //sort by begin time
+    //eventsArrayList could be more specific 
+    //initialize > populate, not initialize > initilaize > populate
```

Comment:
Looks like these are notes on changes you intended to make. Some changes you've made, some you haven't.

Reply:
sorry about that! I'm still working on it

Reply:
ok should be good now. I'm not sure what the unrelated push was